token/core/common/validator.go

```diff
@@ -121,7 +135,7 @@ func (v *Validator[P, T, TA, IA]) VerifyTokenRequest(ledger driver.Ledger, signa
 	for _, action := range ta {
 		actions = append(actions, action)
 	}
-	return actions, nil, nil
+	return actions, attributes, nil
```

Comment:
I wouldn't return the attributes. It is confusing because it seems that this method could modify the map and return it. So tracking what the actual values of the map are becomes difficult (one needs to track all methods where this is passed). I would return only actions and error instead.